wdtk-dumpfiles/src/main/java/org/wikidata/wdtk/dumpfiles/DumpProcessingController.java

```diff
@@ -536,18 +542,23 @@ public MwDumpFile getMostRecentDump(DumpContentType dumpContentType) {
 	 *            the dump file processor to use
 	 */
 	void processDumpFile(MwDumpFile dumpFile,
-			MwDumpFileProcessor dumpFileProcessor) {
+			MwDumpFileProcessor dumpFileProcessor) 
+					throws IOException, FileAlreadyExistsException {
 		try (InputStream inputStream = dumpFile.getDumpFileStream()) {
 			dumpFileProcessor.processDumpFileContents(inputStream, dumpFile);
 		} catch (FileAlreadyExistsException e) {
-			logger.error("Dump file "
+			String errorMessage = "Dump file "
 					+ dumpFile.toString()
 					+ " could not be processed since file "
 					+ e.getFile()
-					+ " already exists. Try deleting the file or dumpfile directory to attempt a new download.");
+					+ " already exists. Try deleting the file or dumpfile directory to attempt a new download.";
+			logger.error(errorMessage);
```

Comment:
I am not sure the logging is still useful now that we raise the exception again.